spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -194,6 +195,13 @@ public EventProcessorInfoConfiguration processorInfoConfiguration(
                                                    c -> configuration);
     }
 
+    @Bean
```

Comment:
I feel that `@ConditionalOnMissingBean` should be added in case somebody else provides a `HeartbeatConfiguration`